src/components/body/About.component.tsx

```diff
@@ -0,0 +1,35 @@
+import redRock from '@/assets/red rock.png';
+
+import styles from './about.module.css';
+export const AboutComponent = () => (
+    <main className={styles.main}>
+        <div className={styles.box}>
+            <div className={styles.boxText}>
+                <h1 className={styles.h1}>About me</h1>
+                <p className={styles.text}>
+                    Hi! My name is Artur and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like
+                    React, HTML, CSS, JavaScript and Git version control system. <br />
+                    <br /> This page was developed during the course{' '}
+                    <a href="https://www.mastersacademy.education/frontend-react-it" target="_blank" rel="noopener noreferrer">
```

Comment:
Considering the repeated use of `<a>` tags with identical functionality and styling throughout the application, it would be beneficial to encapsulate this pattern into a reusable component. 
This approach will centralize common logic and styles, facilitating easier updates and ensuring consistency across the application. Please create a helper component for the `<a>` tags to streamline code maintenance and improve readability.

```tsx
<Link href target ...etc></Link>
```
component should be able accept any props which native `a` tag can accept

Hint: Go to definiton files of a tag you can do it by `ctrl + click` or `cmd + click` on MacOs

![Screenshot 2024-04-19 at 18 10 36](https://github.com/Allocene/fe-react-2024/assets/55930010/e9051de3-4049-4ac1-b776-ab02dbe7ffc2)

Also you link style do not corresponding to designs
![Screenshot 2024-04-19 at 18 11 33](https://github.com/Allocene/fe-react-2024/assets/55930010/54485d61-c43e-4100-8f67-b82b7fb5d0cd)